### ISSUE TITLE
The epoch in compute_losses_with_smpl function is always 0? thus weight_loss_foot_skating is always set to 0

### ISSUE THREAD

**Issue by bowieshi** [sanweiliti/RoHM#28]
Hi. Thanks for your great work. When I tried to train the PoseNet. I found that in `compute_losses_with_smpl` in `posenet.py`. The `epoch` argument is always zero. Because in `training_loop_posenet.py`, this code part
```
    def forward_backward(self, batch, epoch):
        self.mp_trainer.zero_grad()
        t, weights = self.schedule_sampler.sample(batch['motion_repr_clean'].shape[0], dist_util.dev())
        losses, model_output = self.diffusion_train.training_losses(model=self.model, batch=batch, t=t, noise=None, smplx_model=self.smplx_neutral)
        loss = (losses["loss"] * weights).mean()
        self.mp_trainer.backward(loss)
        return losses
```
reference: https://github.com/sanweiliti/RoHM/blob/57ba22cde1595285276ddb88c8c82b25bf0d9722/train/training_loop_posenet.py#L285
The `epoch` is not set in `self.diffusion_train.training_losses(model=self.model, batch=batch, t=t, noise=None, smplx_model=self.smplx_neutral)`. So the `epoch` in `compute_losses_with_smpl` is always default zero. Thus the weight_loss_foot_skating is always set to 0 because `start_skating_loss_epoch` is larger than zero. I wonder is this a bug or just I misunderstood the code. Could you give me some advice or explanation about this. Thank you very much!

**Comment by sanweiliti** [comment]
Hi, thanks for pointing out the issue! It seems that the epoch argument is indeed missing in the function, and foot skating loss has been calculated from the first epoch. However it shouldn't inflence the model performance significantly whether adding it from the first epoch or from later epochs, as it is just a small regularization term.
